docs/sections/getting_started/basics.md

```diff
@@ -2,41 +2,42 @@
 
 # AiiDA basics
 
-This module will give you a first taste of some of the features of AiiDA, and help you familiarize with the `verdi` command-line interface (CLI), as well as AiiDA's IPython shell.
+This module will give you a first taste of some of the features of AiiDA, and help familiarize you with the `verdi` command-line interface (CLI), as well as AiiDA's IPython shell.
 
-```{important}
+::::{tip}
 
-Before starting this tutorial, make sure you have watched the demonstration on [working with your virtual machine](<https://youtu.be/vlmjVwGJgEU>).
-Also remember to run `workon aiida` in any new terminal, in order to enter the correct virtual environment, otherwise the `verdi` command will not be available.
+The `verdi` command supports **tab-completion**!
+In the terminal, type `verdi`, followed by a space and press the 'Tab' key twice to show a list of all the available sub commands.
+Next, try typing `verdi st` and then press "Tab" again.
+This should _tab-complete_ into `verdi status`.
+For help on `verdi` or any of its subcommands, simply append the `--help/-h` flag, e.g.:
 
+```{code-block} console
+$ verdi status -h
 ```
 
-* The `verdi` command supports **tab-completion**:
-  In the terminal, type `verdi`, followed by a space and press the 'Tab' key twice to show a list of all the available sub commands.
-* For help on `verdi` or any of its subcommands, simply append the `--help/-h` flag, e.g.:
-
-  ```
-  $ verdi quicksetup -h
-  ```
+::::
 
-More details on `verdi` can be found in the {ref}`online documentation <aiida:topics:cli>`.
+:::{margin} {{ aiida }} **Further reading**
+More details on the `verdi` CLI can be found in the {ref}`AiiDA documentation <aiida:topics:cli>`.
+:::
```

Comment:
I think this needs to be before the tip box.

Reply:
I put it there first, but then the Contents in the side bar sort of overlap it when you open the page. I didn't think it looked nice, so I moved it here.

Ideally it would be next to the "Tip", but just a bit lower. However, it seems you can't put a `{margin}` directive inside another one.